watertap/data/techno_economic/nanofiltration.yml

```diff
@@ -0,0 +1,92 @@
+default:
+  removal_mass_solute:
+    eeq:
+      constituent_longform: Estradiol Equivalency (EEQ)
+      units: kg/kg
+      value: 0.75
+    endothall:
+      constituent_longform: Endothall
+      units: kg/kg
+      value: 0.8
+    endrin:
+      constituent_longform: Endrin
+      units: kg/kg
+      value: 0.8
+    hardness:
+      constituent_longform: Hardness
+      units: kg/kg
+      value: 0.97
+    hydroxide_alkalinity_as_caco3:
+      constituent_longform: Hydroxide Alkalinity as CaCO3
+      units: kg/kg
+      value: 0.97
+    iron:
+      constituent_longform: Iron
+      units: kg/kg
+      value: 0.8
+    magnesium:
+      constituent_longform: Magnesium, Dissolved
+      units: kg/kg
+      value: 0.99
+    manganese:
+      constituent_longform: Manganese
+      units: kg/kg
+      value: 0.8
+    ndma:
+      constituent_longform: N-nitrosodimethylamine (NDMA)
+      units: kg/kg
+      value: 0.05
+    nonvolatile_toc:
+      constituent_longform: Nonvolatile TOC
+      units: kg/kg
+      value: 0.93
+    perfluorooctane_sulfonate_pfos:
+      constituent_longform: Perfluorooctane sulfonate (PFOS)
+      units: kg/kg
+      value: 0.9
+    perfluorootanoate_pfoa:
+      constituent_longform: Perfluorootanoate (PFOA)
+      units: kg/kg
+      value: 0.9
+    phosphates:
+      constituent_longform: Phosphates, Dissolved
+      units: kg/kg
+      value: 0.99
+    potassium:
+      constituent_longform: Potassium
+      units: kg/kg
+      value: 0.99
+    sodium:
+      constituent_longform: Sodium, Dissolved
+      units: kg/kg
+      value: 0.97
+    sulfate:
+      constituent_longform: Sulfate, Dissolved
+      units: kg/kg
+      value: 0.97
+    sulfur:
+      constituent_longform: Sulfur
+      units: kg/kg
+      value: 0.97
+    toc:
+      constituent_longform: Total Organic Carbon (TOC)
+      units: kg/kg
+      value: 0.75
+    tss:
+      constituent_longform: Total Suspended Solids (TSS)
+      units: kg/kg
+      value: 0.97
+    ultraviolet_transmittance_uvt:
```

Comment:
UV transmittance is more of a feedwater characteristic--not really something that should be under removal_mass_solute